components/layout/table.rs

```diff
@@ -230,7 +230,7 @@ impl Flow for TableFlow {
         // part of the table, which we don't want to do—it belongs to the table wrapper instead.
 
         // Get column inline sizes from colgroups
-        for kid in self.block_flow.base.child_iter().filter(|kid| kid.is_table_colgroup()) {
+        for kid in self.block_flow.base.child_iter_mut().filter(|kid| kid.is_table_colgroup()) {
```

Comment:
Okay, I didn't catch that he did. Two options here, either 1) don't rename this to `child_iter_mut()`, or 2) rename every instance of `child_iter()` into `child_iter_mut()`.

Per discussion on IRC though, both me and @frewsxcv agree that 2) should be taken instead, since it makes everything more consistent.